docs/manage/config-policies.rst

```diff
@@ -0,0 +1,263 @@
+-  **Constraints**: Restrictions that prevent users from exceeding resource limits.
+
+These policies are essential for managing task priorities, resource allocation, and setting specific
+configurations across the organization. Global policies override workspace policies, and both
+override user-submitted configurations.
+
+**************
+ Key Features
+**************
+
+-  Apply policies via WebUI or CLI
+-  Define limits for resource usage, environment settings, and more
+-  Apply policies at different levels (cluster-wide or workspace)
+-  Override capabilities for specific user roles or groups
+-  Set different priority limits for experiments and NTSC (notebooks, TensorBoards, shells, and
+   commands) tasks
+-  Allow priority overrides within policy constraints
+
+******************
+ Setting Policies
+******************
+
+Administrators can set Config Policies at either the cluster or workspace levels through the WebUI
+or CLI.
+
+WebUI
+=====
+
+Administrators can set Config Policies at both the cluster and workspace levels.
+
+.. tabs::
+
+   .. tab::
+
+      Cluster Config Policy
+
+      To set configuration policies at the cluster level:
+
+      #. Sign in to the Determined WebUI as a cluster administrator.
+      #. Navigate to **Config Policies**.
+      #. Choose **Experiments** or **Tasks** to display the editable configuration file.
+      #. Define the policies, then click **Apply**.
+      #. Confirm you want to apply the policies.
+
+   .. tab::
+
+      Workspace Config Policy
+
+      To set configuration policies at the workspace level:
+
+      #. Sign in to the Determined WebUI as a cluster or workspace administrator.
+      #. From your workspace, navigate to the **Config Policies** tab.
+      #. Choose **Experiments** or **Tasks** to display the editable configuration file.
+      #. Define the policies, then click **Apply**.
+      #. Confirm you want to apply the policies.
+
+CLI
+===
+
+Use the following commands to manage Task Config Policies via CLI:
+
+.. code:: bash
+
+   # List existing policies
+   det policy list
+
+   # Create a new policy
+   det policy create <policy_name> --config <policy_config_file.yaml>
+
+   # Update an existing policy
+   det policy update <policy_name> --config <updated_policy_config_file.yaml>
+
+   # Delete a policy
+   det policy delete <policy_name>
+
+*****************
+ Policy Examples
+*****************
+
+Below are some examples of Config Policies to help guide administrators.
+
+Limiting Resources
+==================
+
+Administrators can set constraints on resource usage, allowing them to manage how users allocate
+resources for workloads such as experiments, notebooks, tensorboards, shells, and commands. The two
+main configurable constraints are:
+
+-  ``resources.max_slots``: Limits the maximum number of slots (GPUs or CPUs) that can be used.
+-  ``resources.priority_limit``: Sets the priority limit for tasks.
```

Comment:
I'm so sorry @tara-det-ai - we decided to keep `priority_limit` under `constraints` after all. So it's `constraints.priority_limit` and `constraints.resources.max_slots`. (This was decided EOD yesterday, very last minute)

Looking at the time we have left for the project, we felt it would be best spent on bug fixes, and that this change wasn't worth the risk at this stage.

Reply:
thank you!